Dribbblish/color.ini

```diff
@@ -125,6 +125,25 @@ notification       = ff9e00
 notification-error = f61379
 misc               = DEDEDE
 
+[konai]

```

Comment:
Please add a screenshot to Dribbblish's README and THEMES.md

Reply:
@Kerago can you do this?